docs/get-started.asciidoc

```diff
@@ -0,0 +1,299 @@
+////
+Goal of this doc:
+The user is able to successfully see data from their .NET application make it to the Elastic UI via the Elastic Distribution for OpenTelemetry .NET
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+* They have dotnet installed
+////
+
+[[get-started]]
+== Get started
+
+:language: .NET
+:language_lc: dotnet
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+This guide shows you how to use the {distro_name} ("the distro") to instrument your {language} application and send OpenTelemetry data to an Elastic Observability deployment.
+
+////
+Is this true?
```

Comment:
Is what true? Sorry not sure what this question pertains

Reply:
Is this true and it is something we want to include in the docs?

>As an OpenTelemetry SDK, the distro supports sending data to any OpenTelemetry protocol (OTLP) endpoint ([OpenTelemetry Collector](https://opentelemetry.io/docs/collector/)) [but this guide assumes you are sending data to Elastic].

(Context: This came up while I was standardizing the docs across languages. It was mentioned in the Node.js docs.  I'm wondering if this is true of all the distros _and_ if it's worth mentioning.

Reply:
This is true and love the addition 👍